### DISCUSSION TITLE
Basics

### DISCUSSION THREAD

**Discussion by pilvijo** [Planetable/Planet#152] [Q&A]
Hi, 
I am quite new with Planet and I would like to be sure how the build the basics in website. 
Is it possible to create a walkthrough with snapshots. Actually I can create one for future needs, if I can get some help with my first steps. 

Manual instructions could maybe be integrated in the website later on. Could it be an idea for a hackathon?

In the snapshot there is one of the sheets, just for example.

Thanks already!

<img width="491" alt="Näyttökuva 2023-1-4 kello 16 17 06" src="https://user-images.githubusercontent.com/103053740/210574681-17a9fd50-726f-45c4-89b2-e12de5672348.png">

**Comment by livid** [comment]
Hi @pilvijo 

Thank you for the feedback! As we are adding more features, they do need better guidance for the users.

For the analytics feature, it was previously introduced in a blog post here:

https://planetable.eth.limo/C171164A-F0B7-4915-9FC6-4BC4217D7971/

But in the UI, we should also add a link to the docs. Thanks again!

Other docs are available here:

https://www.planetable.xyz/guides/

The guides website is also an open-source repo:

https://github.com/Planetable/PlanetableXYZWebsite/tree/main/content/guides

**Comment by pilvijo** [comment]
Thank you very much!Love to start building with you and this tool. Hopefully my actions lead to interest arising herearound! PilviLähetetty iPhonestaXin Liu ***@***.***> kirjoitti 11.1.2023 kello 8.18:﻿
Hi @pilvijo
Thank you for the feedback! As we are adding more features, they do need better guidance for the users.
For the analytics feature, it was previously introduced in a blog post here:
https://planetable.eth.limo/C171164A-F0B7-4915-9FC6-4BC4217D7971/
But in the UI, we should also add a link to the docs. Thanks again!
Other docs are available here:
https://www.planetable.xyz/guides/
The guides website is also an open-source repo:
https://github.com/Planetable/PlanetableXYZWebsite/tree/main/content/guides
